platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -53,6 +53,23 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="config-cache"></a>
+### Configuration cache improvements
+
+The [configuration cache](userguide/configuration_cache.html) improves build performance by caching the result of the configuration phase. Gradle uses the configuration cache to skip the configuration phase entirely when nothing that affects the build configuration has changed.
+
+#### Parallel caching for faster loading times
```

Comment:
🤔  This section suggests that parallel CC is "free lunch", but we know that it can have undesired effects on the build, including build failures. It would be better if we at least mention this to encourage people to read the details in the docs before enabling it.